bayesian_cnn_prometheus/tools/results_visualizer.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+from bayesian_cnn_prometheus.constants import Paths
+from bayesian_cnn_prometheus.evaluation.utils import load_nifti_file, get_standardized_slice, save_as_nifti
+
+
+class ResultsVisualizer:
+    def __init__(self):
+        ...
+
+    def visualize_patient_results(self, patient_id: str, predictions: np.array, slice_number: int,
+                                  save_variance: bool = False):
+        image_file_path = str(Paths.IMAGE_FILE_PATTERN_PATH).format(patient_id, 'nii.gz')
+        mask_file_path = str(Paths.MASK_FILE_PATTERN_PATH).format(patient_id, 'nii.gz')
+        reference_file_path = str(Paths.REFERENCE_SEGMENTATION_FILE_PATTERN_PATH).format(patient_id, 'nii.gz')
+
+        image = load_nifti_file(image_file_path)
+        mask = load_nifti_file(mask_file_path)
+        reference = load_nifti_file(reference_file_path)
+
+        segmentation_from_mean = self.get_segmentation_from_mean(predictions)
+        segmentation_variance = self.get_segmentation_variance(predictions)
+
+        if save_variance:
+            segmentation_variance_path = Paths.RESULTS_PATH / Path(f'SEGMENTATION_VARIANCE_{patient_id}.nii.gz')
+            save_as_nifti(segmentation_variance, segmentation_variance_path)
+
```

Comment:
Dodałam coś takiego, żeby można było przekazać tą wariancję do narzędzia analizującego korelację jako plik. Jak już będzie to narzędzie gotowe to zmienimy, żeby ta wariancja była we flow analizy od razu przekazywana tam :)